MAINTAINERS.md

```diff
@@ -1,3 +1,11 @@
-# MAINTAINERS
+Trestle demos was designed and open sourced by a team based at [IBM Research](https://www.research.ibm.com/) and others around the world.  The list includes:
```

Comment:
Do we need this maintainer prefix if the attribution information is already in the README?

Reply:
File is now identical to that of compliance-trestle.